pkg/scheduler/scheduler_test.go

```diff
@@ -1200,3 +1197,122 @@ func TestSchedulerBinding(t *testing.T) {
 		})
 	}
 }
+
+// TestInjectingPluginConfigForVolumeBinding tests injecting
+// KubeSchedulerConfiguration.BindTimeoutSeconds as args for VolumeBinding if
+// no plugin args is configured for it.
+// TODO remove when KubeSchedulerConfiguration.BindTimeoutSeconds is eliminated
+func TestInjectingPluginConfigForVolumeBinding(t *testing.T) {
+	policyAlgorithmSC := schedulerapi.SchedulerAlgorithmSource{
+		Policy: &config.SchedulerPolicySource{},
```

Comment:
Rather, we should be testing with Provider.

Reply:
Oops, this is used down below.
Just inline values, specially if they are only used once.

Reply:
updated